### ISSUE TITLE
[Feature] Update service

### ISSUE THREAD

**Issue by SirWindfield** [Drombler/drombler-fx#116]
Some update service would be nice to have. The fact that you already use OSGI would make this quite easy to implement.

**Comment by puce77** [comment]
Agreed, this would be a great feature. 

Another approach would be using the app stores of the various platforms:

- Mac: [Mac App Store](http://www.apple.com/osx/apps/app-store/)
- Windows 10: [Windows Store](https://www.microsoft.com/en-us/windows/apps-and-games)
- Ubuntu: [PPA](https://launchpad.net/ubuntu/+ppas)s

**Comment by SirWindfield** [comment]
I think so. One small question which is kind of off topic :) I started to implement an application using the mvvmfx library. It is really nice and easy to use but I think yours is more powerful for the small amount of extra configuration (annotations) that are needed. I am facing a design problem. If I have for example on the left dockable view a list view that contains some polo's, how do i notify the center which one is selected? Also how exactly do I retrieve OSGI services within for example dockable views? I would really appreciate your help!

**Comment by puce77** [comment]
Yes, I think we should have some update mechanism in addition of app stores.

Regarding you questions: could you file a new question issue or, maybe even better, ask them on StackOverflow using the drombler-fx tag: http://stackoverflow.com/questions/tagged/drombler-fx
And split the question in 2 questions (selection and OSGi service).

**Comment by SirWindfield** [comment]
Ok. Thank you!

**Comment by SirWindfield** [comment]
Link to a JavaFX launcher that is capable of updating its content.
https://github.com/edvin/fxlauncher

**Comment by puce77** [comment]
Issue #137 is currently in progress, which should allow to create installers for Mac and Linux App Stores, at least.
